content/post/087-deprecating-the-old-docs-site.md

```diff
@@ -0,0 +1,51 @@
+---
+date: 2020-04-01
+title: Deprecating the old docs site
+author: Johnny Matthews
+---
+
+# Deprecating the old docs site
+
+The IPFS team is deprecating the old documentation website. That's just a fancy way of saying we're replacing the current docs.ipfs.io website with the new docs-beta.ipfs.io website.
+
+## We built a new docs site
+
+Work on a new documentation site for IPFS started in Q3 of 2019. Before this, the docs site was built upon Hugo and hosted by Netlify. While Hugo is an excellent static-site generator, it lacks an encapsulated component model, relies upon old web development conventions, and requires DOM manipulation if you want to have any kind of site interactivity. Similarly, Netlify is an incredible hosting solution. However, the centralized nature of the service goes against our mission here at Protocol Labs.
```

Comment:
The legacy docs actually weren't hosted on Netlify -- we were just using Netlify as a stand-in for hosting/deploying the docs beta while we wrapped up figuring out IPFS hosting. I'd just get rid of the whole Netlify part. Maybe add some thoughts around it just not being as easy to build on, particularly if not everyone in your org is conversant in Go. But in any case, I'd run all this past @cwaring for his blessing too.

Reply:
Ah ok, thought the legacy docs were still on Netlify. I'll take that part out then.